index.html

```diff
@@ -64,7 +64,7 @@ <h4>i am a software developer</h4>
     <section class="work-section">
       <h1>My Recent Works</h1>
       <div class="img-container">
-        <img src="./images/Img Placeholder(9).png" alt="" />
+        <img src="./images/Img Placeholder(9).png" alt="My recent work image" />
         <div class="work-text">
           <h2>Multi-Posts Stories</h2>
           <p>
```

Comment:
- [ ] I highlighted this because I am not able to see your `navlist`.. I suggest that elements of your navigation list should take you to their section let us say: when I click `contact` it should take you to the contact section, same as `about` etc. make sure each link lead you to somewhere